index.html

```diff
@@ -70,14 +72,15 @@ <h3 class="connectlink">LET´S CONNECT</h3>
           </div>
         </div>
       </div>
+    </div>
     </section>
 
     <section class="works">
```

Comment:
- [ ] Kindly fix the work section
![Captureandh](https://user-images.githubusercontent.com/81444338/173162658-4f52a46d-9b1d-4f0b-8d16-83ae74c533d3.PNG)

Reply:
- [ ] As the previous reviewer mentioned, please fix your design to math Figma design

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/173189924-4e642af9-a58a-4ed6-b6ab-03d17e40086d.png)  | ![image](https://user-images.githubusercontent.com/53402600/173189948-4b1797fb-c085-407d-b36f-ea087bdb51ca.png) |